tests/Tests/500__jupyterhub/test-filling-pvc.robot

```diff
@@ -0,0 +1,38 @@
+*** Settings ***
+Resource            ../../Resources/ODS.robot
+Resource            ../../Resources/Common.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library             DebugLibrary
+
+Suite Setup         Server Setup
+Suite Teardown      End Web Test
+
+
+*** Variables ***
+${PATH_TO_FILE}         ods-ci-notebooks-main/notebooks/200__monitor_and_manage/203__alerts/notebook-pvc-usage/fill-notebook-pvc-to-complete-100.ipynb
+${LINK_OF_GITHUB}       https://github.com/redhat-rhods-qe/ods-ci-notebooks-main.git
+${FILE_NAME}            fill-notebook-pvc-to-complete-100.ipynb
+
+
+*** Test Cases ***
+Get Notifications If Storage Capacity Limits Get Exceeded
```

Comment:
Could you consider renaming it to `Verify Users Get Notifications If Storage Capacity Limits Get Exceeded` ?

Reply:
yes I will do it